das/stats.go

```diff
@@ -22,9 +22,10 @@ type SamplingStats struct {
 }
 
 type WorkerStats struct {
-	Curr uint64 `json:"current"`
-	From uint64 `json:"from"`
-	To   uint64 `json:"to"`
+	JobType jobType `json:"job_type"`
```

Comment:
Please update api/docgen/exampledata/samplingstats to include this - you can just run `celestia rpc das SamplingStats | jq` on a running node and paste the output into that file

Reply:
this is my only blocking comment by the way, the rest are optional

Reply:
TIL. thank you!